handbook/rbac/rbac.md

```diff
@@ -0,0 +1,128 @@
+# Workspaces and Role-based Access Control
+
+In SystemLink 2020R2 NI introduced a new role-based access control (RBAC) system. This security technology provides strong isolation between different *workspaces* as well as fine grain privileges for systems, data, and analysis routines in SystemLink. Collectively this capability allows access to SystemLink to scale user and data access beyond a single team to an entire organization.
+
+## Concepts and Design
+
+SystemLink RBAC leverages common concepts such as *roles* and *users*. It also introduces concepts such as *workspaces* and *automatic data encapsulation* to ensure strict access control of data produced by test systems.
+
+<figure>
+    <img src="../../img/rbac-concepts.png" width="500" />
+    <figcaption>Role-based access control concepts in SystemLink</figcaption>
+</figure>
+
+### Workspace
+
+A workspace is a complete encapsulation of all systems, data, and other resources within SystemLink. If a user is not a member of a given workspace they cannot access or modify any of the resources within the workspace. This is true for access via the SystemLink web application and the SystemLink REST API. The SystemLink Advanced Server license is required to create multiple workspaces, but there is no limit on the number of workspaces that can be created. If a user is a member of multiple workspaces resource from each workspace will be shown together within the grids, dashboards, and other data views throughout SystemLink.
```

Comment:
"If a user is a member of multiple workspaces resource from each workspace"
This reads weird

Reply:
I cleaned up the wording and added a comma that should help it read better.